account_cryptocurrency/models/res_currency_move.py

```diff
@@ -0,0 +1,231 @@
+# Copyright 2021 ForgeFlow S.L.
+# Copyright 2018 Fork Sand Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+_STATES = [
+    ("draft", "Draft"),
+    ("posted", "Posted"),
+    ("cancel", "Cancelled"),
```

Comment:
![Selection_188](https://user-images.githubusercontent.com/25005517/132020197-5ed69a3b-b722-4d19-9590-14164b34bc51.png)

migration scripts are needed for this change

Reply:
Nice catch, added them :+1: